vetj00-sp2

vetrovskyj opened the pull request:
Link to application: https://eso.vse.cz/~vetj00/sp2/

vetrovskyj committed:
vetj00-sp2

Link to application: https://eso.vse.cz/~vetj00/sp2/

vetrovskyj committed:
Add files via upload

nvbach91 commented on www/vetj00/sp2/assets/js/article.js:
```diff
@@ -0,0 +1,75 @@
+                console.log(headingSource);
+                const section = $(`<section></section>`);
+                const headingAndContent = $(`<div class="heading-and-content"></div>`);
+                const heading = $(`<a href="" onclick="openFullArticle();" class="heading-link">${headingSource}</a>`);
```
nesmite pouzivat inline js

nvbach91 reviewed (changes requested):
- opravit for cykly
- jednotlive moduly je potreba sloucit do jedne stranky (napr. pri klikani na articles se stranka ne refreshne)
- prispevky defaultne seradit od nejnovejsiho
- pridavani prispevku obcas nefunguje
- opravit opakovani aktualizace DOMu (stranka articles)

vetrovskyj committed:
Delete www/vetj00/sp2/articles-full directory

vetrovskyj committed:
Delete www/vetj00/sp2/articles directory

vetrovskyj committed:
Delete article.js

vetrovskyj committed:
Delete articles-full.js

vetrovskyj committed:
Oprava SP2

https://eso.vse.cz/~vetj00/sp2/
Opravené cykly
Vše sloučeno do single-page
Příspěvky jsou řazené od nejnověji přidaného
Opraveno opakování aktualizace DOMu

nvbach91 reviewed (approved):
![how-to-create-presentation-slides-that-are-out-of-this-world-by-slidecomet-itseugenec-kaixinspeaking-28-638](https://user-images.githubusercontent.com/20724910/104008962-8b66ea80-51aa-11eb-9b47-d64347f9797a.jpg)
